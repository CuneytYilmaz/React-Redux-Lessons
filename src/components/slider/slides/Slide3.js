import React from 'react';

import Button from '../../ui/Button';

import image from '../../../img/slides/slider-image-3.svg';

const Slide3 = () =>
  <div className="row Slider__slide">
    <div className="col-md-6">
      <h1 className="Slider__title Slider__title--main">
        Yağlı ellerinle klavyeye dokunma, seslenerek Tepsi’yi kontrol et!
      </h1>
      <p className="u-gap-bottom">
        Yağlı ellerinle klavye ve mouse’unu kirletme, seslenerek Tepsi’ye komutlar ver. Hangi
        komutların ne işe yaradığını öğrenmek için buraya tıkla!
      </p>
      <Button>Kullanmaya Başla!</Button>
    </div>
    <div className="col-md-6 u-text-center">
      <img
        src={image}
        className="u-inline-block"
        alt="Yağlı ellerinle klavyeye dokunma, seslenerek Tepsi’yi kontrol et!"
        width="400"
        height="378"
      />
    </div>
  </div>;

export default Slide3;
