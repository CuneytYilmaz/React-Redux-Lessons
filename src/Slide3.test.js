import React from 'react';
import { shallow } from 'enzyme';

import Slide3 from './components/slider/slides/Slide3';

it('renders correctly', () => {
  const wrapper = shallow(<Slide3 />);

  expect(wrapper).toMatchSnapshot();
});
