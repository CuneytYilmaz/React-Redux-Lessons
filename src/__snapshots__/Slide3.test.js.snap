// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row Slider__slide"
>
  <div
    className="col-md-6"
  >
    <h1
      className="Slider__title Slider__title--main"
    >
      Yağlı ellerinle klavyeye dokunma, seslenerek Tepsi’yi kontrol et!
    </h1>
    <p
      className="u-gap-bottom"
    >
      Yağlı ellerinle klavye ve mouse’unu kirletme, seslenerek Tepsi’ye komutlar ver. Hangi komutların ne işe yaradığını öğrenmek için buraya tıkla!
    </p>
    <Button
      primary={true}
      secondary={false}
    >
      Kullanmaya Başla!
    </Button>
  </div>
  <div
    className="col-md-6 u-text-center"
  >
    <img
      alt="Yağlı ellerinle klavyeye dokunma, seslenerek Tepsi’yi kontrol et!"
      className="u-inline-block"
      height="378"
      src="slider-image-3.svg"
      width="400"
    />
  </div>
</div>
`;
